src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerIT.java

```diff
@@ -39,11 +54,19 @@ protected boolean preserveClusterUponCompletion() {
         return true;
     }
 
-    // This method is the same as ESRestTestCase#buildClient, but it attempts to connect
-    // to the provided cluster for 1 minute before giving up. This is useful when we spin up
-    // our own Docker cluster on the local node for Integration Testing
+    protected boolean isHttps() {
+        return System.getProperty("tests.https", "false").toLowerCase().equals("true");
```

Comment:
Can we move all the `configs` such as this into a class of its own and have documentation saying the value options they take and what they mean and the defaults ?

Reply:
Done